tensorflow_datasets/scripts/benchmark_dataset.py

```diff
@@ -0,0 +1,134 @@
+r"""Script to Benchmark dataset.
+
+Standalone script to benchmark dataset.
+The script by default runs in graph mode which gives us better performance.
+
+Instructions:
+
+```
+python -m tensorflow_datasets.scripts.benchmark_dataset \
+  --dataset dataset_name \
+  --epochs 3
+```
+
+Use the `--help` flag for more options and details.
+
+"""
+import time
+from typing import Dict, Tuple
+
+from absl import app
+from absl import flags
+from absl import logging
+from packaging import version
+
+import tensorflow.compat.v2 as tf
+import tensorflow_datasets as tfds
+
+NUM_EXECUTIONS = 1
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string('dataset', None, 'Dataset name')
+flags.DEFINE_string(
+    'data_dir', None,
+    'Path to dataset directory (auto-computed). Dataset should already present')
+flags.DEFINE_integer(
+    'epochs', NUM_EXECUTIONS,
+    'No. of iterations over the dataset to average out performance')
+flags.DEFINE_integer(
+    'num_examples', None,
+    'No. of examples of Dataset to be used for Benchmarking Dataset.'
+    'Defaults to size of Train split')
+flags.DEFINE_boolean('disable_tqdm', False, 'If True, disable progress bar.')
+flags.DEFINE_boolean('enable_eager_exec', False,
+                     'If True, enable eager execution.')
+
+
+def _make_ds(
+    name: str,
+    data_dir: str,
+) -> Tuple[Dict[str, tf.data.Dataset], int]:
+  """Download, prepare and preprocess Dataset"""
+
+  builder = tfds.builder(name, data_dir=data_dir)
+
+  if FLAGS.num_examples is None or (
+      FLAGS.num_examples >= builder.info.splits['train'].num_examples):
+    num_examples = builder.info.splits['train'].num_examples
+  else:
+    num_examples = FLAGS.num_examples
+
+  # Exclude dataset download time
+  builder.download_and_prepare()
+
+  percentage = int(
+      num_examples * 100 / builder.info.splits['train'].num_examples)
+  start_time = time.perf_counter()
+  ds = builder.as_dataset(split=f'train[:{percentage}%]')
+  print('\nTime taken to preprocess {} dataset: {}\n'
+        .format(name, time.perf_counter() - start_time))
+
+  return ds, num_examples
+
+@tf.function
+def benchmark(
+    ds: tf.data.Dataset,
+    epochs: int,
```

Comment:
When renaming `EPOCH` -> `NUM_EXECUTION`, all instance of `epoch` should be renamed. Even if I only post a single review comment. I assume that the fix is applied globally.